chapters/chapter-12/application/build.gradle

```diff
@@ -28,14 +28,14 @@ dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
   implementation 'io.awspring.cloud:spring-cloud-aws-starter-ses'
   implementation 'io.awspring.cloud:spring-cloud-aws-starter-sqs'
+  implementation 'org.flywaydb:flyway-core'
   implementation "org.thymeleaf.extras:thymeleaf-extras-springsecurity6:${thymeleafExtrasSpringSecurityVersion}"
   implementation 'software.amazon.awssdk:cognitoidentityprovider'
   implementation 'com.google.code.findbugs:jsr305:3.0.2'
   implementation 'org.webjars:webjars-locator-core'
   implementation 'nz.net.ultraq.thymeleaf:thymeleaf-layout-dialect'
   implementation 'org.webjars:bootstrap:4.6.1'
   implementation 'org.webjars:font-awesome:5.15.3'
-  developmentOnly 'org.springframework.boot:spring-boot-devtools'
```

Comment:
What's the exact issue here? We should still be able to use the Spring Boot DevTools.